app/webpacker/css/admin/products_v3.scss

```
@@ -36,6 +60,12 @@
     padding: 4px;
     border-collapse: separate; // This is needed for the outer padding. Also should be helpful to give more flexibility of borders between rows.
 
+    thead {
+      position: sticky;
+      top: 0;
+      z-index: 999;
```

I'd like to avoid arbitrary large numbers. Would a value of 1 work, like on .form-actions?

```suggestion
      z-index: 1;
```